README.fr.md

```diff
@@ -0,0 +1,122 @@
+étiquetages (`amenity = restaurant`).
+
+La construction d'un index de nom canonique permet deux choses très utiles:
+- Nous pouvons suggérer la manière la plus "correcte" d'étiqueter les choses lorsque les utilisateurs les créent pendant l'édition.
+- Nous pouvons analyser les données OSM pour les fonctionnalités "incorrectes" et produire des listes pour révision et nettoyage.
+
+<img width="1017px" alt="Index de suggestion de noms utilisé dans iD" src="https://raw.githubusercontent.com/osmlab/name-suggestion-index/master/docs/img/nsi-in-iD.gif"/>
+
+* L'index de suggestion de nom est utilisé dans iD lors de l'ajout d'un nouvel élément *
+
+Actuellement utilisé dans:
+* iD (voir ci-dessus)
+* [Vespucci](http://vespucci.io/tutorials/name_suggestions/)
+* Présélections JOSM disponibles
+
+
+### Participer!
+
+* Lisez le projet [Code de conduite](CODE_OF_CONDUCT.md) et n'oubliez pas d'être gentils les uns envers les autres.
+* Voir [CONTRIBUTING.md](CONTRIBUTING.md) pour savoir comment contribuer à cet index.
+
+Nous cherchons toujours de l'aide! Si vous avez des questions ou souhaitez contacter un responsable, envoyez une requête ping à `bhousel` sur:
+* [OpenStreetMap US Slack](https://slack.openstreetmap.us/)
+(Chaînes `# poi` ou` # general`)
+
+
+### Conditions préalables
+
+* [Node.js](https://nodejs.org/) version 8 ou plus récente
+* [`git`](https://www.atlassian.com/git/tutorials/install-git/) pour votre plateforme
+
+
+### Installation
+
+* Clonez ce projet, par exemple:
+  `git clone git@github.com: osmlab / nom-suggestion-index.git»
+* `cd` dans le dossier du projet,
+* Exécutez `npm install` pour installer les bibliothèques
+### À propos de l'index
+#### Fichiers générés (ne pas modifier):
+Fichiers prédéfinis (utilisés par les éditeurs OSM):
+* `dist / name-suggestions.json` - Préréglages de suggestion de nom
+* `dist / name-suggestions.min.json` - Préréglages de suggestion de nom, minifiés
+* `dist / name-suggestions.presets.xml` - Préréglages de suggestion de nom, comme XML prédéfini de style JOSM
+Listes de noms:
+* `dist / names_all.json` - tous les noms et balises fréquents collectés depuis OpenStreetMap
+* `dist / names_discard.json` - sous-ensemble de` names_all` que nous éliminons
+* `dist / names_keep.json` - sous-ensemble de` names_all` que nous conservons
+* `dist / wikidata.json` - données de marque en cache récupérées à partir de Wikidata
+#### Fichiers de configuration (modifiez-les):
+* `config / *`
+  * `config / filters.json` - Expressions régulières utilisées pour filtrer` names_all` dans `names_keep` /` names_discard`
+* `marques / *` - Fichiers de configuration pour chaque type d'entreprise de marque, organisés par balise OpenStreetMap
+  * `marques / équipement / *. json`
+  * `marques / loisirs / *. json`
+  * `marques / boutique / *. json`
+  * `marques / tourisme / *. json`
+  * `marques / bureau / *. json`
+
```

Comment:
these are files / paths so must not be translated and must not have spaces in-between.